packages/vuepress-theme-odyssey/global-components/Anatomy.vue

```diff
@@ -1,6 +1,14 @@
 <template>
-  <figure v-if="img" class="docskit-figure-anatomy">
-    <img :src="img" :alt="alt" />
+  <figure v-if="img" 
+      :class="{
+      'docskit-figure-anatomy': true,
+      [`is-figure-anatomy-filled`]: fill
+    }"
```

Comment:
style-nit: Spacing

```suggestion
      :class="{
        'docskit-figure-anatomy': true,
        [`is-figure-anatomy-filled`]: fill
      }"
```

Reply:
Thanks for catching this, I was relying on eslint to catch this sort of stuff. Ill take another look at the config.